test/unittesting/testingutils.go

```diff
@@ -0,0 +1,624 @@
+			}
+			if strings.Split(strings.Split(cookie, ";")[1], "=")[0] == " Path" {
+			}
+			for _, property := range strings.Split(cookie, ";") {
+				if strings.Index(property, "HttpOnly") == 1 {
+					refreshTokenHttpOnly = "true"
+					break
+				}
+			}
+		} else if strings.Split(strings.Split(cookie, ";")[0], "=")[0] == "sIdRefreshToken" {
+			idRefreshTokenFromCookie = strings.Split(strings.Split(cookie, ";")[0], "=")[1]
+			if strings.Split(strings.Split(cookie, ";")[2], "=")[0] == " Expires" {
+				idRefreshTokenExpiry = strings.Split(strings.Split(cookie, ";")[2], "=")[1]
+			} else if strings.Split(strings.Split(cookie, ";")[2], "=")[0] == " expires" {
+				idRefreshTokenExpiry = strings.Split(strings.Split(cookie, ";")[2], "=")[1]
+			} else {
+				idRefreshTokenExpiry = strings.Split(strings.Split(cookie, ";")[3], "=")[1]
+			}
+			if strings.Split(strings.Split(cookie, ";")[1], "=")[0] == " Path" {
+			}
+			for _, property := range strings.Split(cookie, ";") {
+				if strings.Index(property, "HttpOnly") == 1 {
+					idRefreshTokenHttpOnly = "true"
+					break
+				}
+			}
+		} else if strings.Split(strings.Split(cookie, ";")[0], "=")[0] == "sAccessToken" {
+			accessToken = strings.Split(strings.Split(cookie, ";")[0], "=")[1]
+			if strings.Split(strings.Split(cookie, ";")[2], "=")[0] == " Expires" {
+				accessTokenExpiry = strings.Split(strings.Split(cookie, ";")[2], "=")[1]
+			} else if strings.Split(strings.Split(cookie, ";")[2], "=")[0] == " expires" {
+				accessTokenExpiry = strings.Split(strings.Split(cookie, ";")[2], "=")[1]
+			} else {
+				accessTokenExpiry = strings.Split(strings.Split(cookie, ";")[3], "=")[1]
+			}
+			if strings.Split(strings.Split(cookie, ";")[1], "=")[0] == " Path" {
+			}
+			for _, property := range strings.Split(cookie, ";") {
+				if strings.Index(property, "HttpOnly") == 1 {
+					accessTokenHttpOnly = "true"
+					break
+				}
+			}
+		}
+	}
+	return map[string]string{
+		"sAccessToken":           accessToken,
+		"sRefreshToken":          refreshToken,
+		"sIdRefreshToken":        idRefreshTokenFromCookie,
+		"refreshTokenExpiry":     refreshTokenExpiry,
+		"refreshTokenDomain":     refreshTokenDomain,
+		"refreshTokenHttpOnly":   refreshTokenHttpOnly,
+		"idRefreshTokenExpiry":   idRefreshTokenExpiry,
+		"idRefreshTokenDomain":   idRefreshTokenDomain,
+		"idRefreshTokenHttpOnly": idRefreshTokenHttpOnly,
+		"accessTokenExpiry":      accessTokenExpiry,
+		"accessTokenDomain":      accessTokenDomain,
+		"accessTokenHttpOnly":    accessTokenHttpOnly,
+	}
+}
+
+func getInstallationDir() string {
+	installationDir := os.Getenv("INSTALL_DIR")
+	if installationDir == "" {
+		installationDir = returnNumberOfDirsToGoUpFromCurrentWorkingDir(false) + "supertokens-root"
+	} else {
+		installationDir = returnNumberOfDirsToGoUpFromCurrentWorkingDir(true) + installationDir
+	}
+	return installationDir
+}
+
+func SetKeyValueInConfig(key string, value string) {
```

Comment:
This function is not being used anywhere. Shouldn't this be used in the tests?

Reply:
@rtpa25 please reply to this..

Reply:
Hey it is actually being used inside authFlow_Test file, it is being used in the emailExistsAndVerificationCheck_test file, it is also being used in the session_test file